operator/pkg/securedcluster/values/translation/translation.go

```diff
@@ -305,21 +306,13 @@ func (t Translator) getCollectorContainerValues(collectorContainerSpec *platform
 		// Override the helm-charts default collectionMethod selection logic.
 		cv.SetBoolValue("forceCollectionMethod", true)
 		switch *c {
-		case platform.CollectionEBPF:
-			// EBPF collection has been deprecated, translate it to CORE_BPF if it's not forced
-			if collectorContainerSpec.ForceCollection != nil &&
-				*collectorContainerSpec.ForceCollection {
-				cv.SetStringValue("collectionMethod", storage.CollectionMethod_EBPF.String())
-			} else {
-				cv.SetStringValue("collectionMethod", storage.CollectionMethod_CORE_BPF.String())
-			}
 		case platform.CollectionNone:
 			cv.SetStringValue("collectionMethod", storage.CollectionMethod_NO_COLLECTION.String())
 		case platform.CollectionCOREBPF:
 			cv.SetStringValue("collectionMethod", storage.CollectionMethod_CORE_BPF.String())
-		case legacyCollectionKernelModule:
-			// Kernel module collection has been removed, but for the
-			// purposes of upgrades, we translate it to CORE_BPF
+		case legacyCollectionKernelModule, legacyCollectionEBPF:
+			// Kernel module and eBPF collection have been removed, but for the
+			// purposes of upgrades, we translate them to CORE_BPF
```

Comment:
Q: Why is it that we now skip any `ForceCollection` related checks in the new version of this file? We are replacing the `ForceCollection`-aware logic with logic which unconditionally transforms `EBPF` to `CORE_BPF`, no?

Reply:
Yes, force collection is deprecated now as well because it only affected EBPF configuration, and now that we have no EBPF drivers in 4.5 we need to unconditionally translate to CORE_BPF

Reply:
thanks